Fix: Formatting issues in docs

yo1995 opened the pull request:
With #919  merged, this PR runs automated check on docs and fixed minor issues.

Will either setup sanity checks in our repo in next release, or periodically check docs formatting in the future.

vquach2404 commented on arcgis-ios-sdk-samples/Edit data/Edit features with feature-linked annotation/README.metadata.json:
```diff
@@ -1,6 +1,6 @@
 {
     "category": "Edit data",
-    "description": "Edit feature attributes which are linked to annotation through an expression.",
+    "description": "Edit feature attributes which are linked to annotations through an expression.",
```
If you're going to make this change, please also update the description in content plist.

yo1995 replied:
Ahh really great catch! So which way should we go? 
In README it is "annotations" while in metadata and content plist it is "annotation".

philium replied:
@vquach2404 Good catch. I just assumed this was fixing a discrepancy between the two.

vquach2404 replied:
@yo1995 let's just change the metadata and plist to "annotations." I originally adjusted the README to use "annotations" instead.

yo1995 commented:
👆  Thanks for pointing this out! I'll add a check to the script to ensure the description strings are also consistent.

yo1995 commented:
I've added the check and fixed more inconsistent plist descriptions! 
Except a few made inconsistent on purpose, now all the descriptions are the same in README, plist and metadata.
